lib/pdk/util/version.rb

```diff
@@ -0,0 +1,34 @@
+require 'pdk/version'
+require 'pdk/cli/exec'
+
+module PDK
+  module Util
+    module Version
+      def self.version_string
+        "#{PDK::VERSION} #{pdk_ref}".strip.freeze
+      end
+
+      def self.pdk_ref
+        ref = pkg_sha || git_ref
+        ref.nil? ? nil : "(#{ref})"
+      end
+
+      def self.pkg_sha
+        version_file = File.join(File.expand_path('../../..', File.dirname(__FILE__)), 'VERSION')
+
+        if File.exist? version_file
+          ver = File.read(version_file)
+          sha = ver.strip.split('.')[-1] unless ver.nil?
+        end
+
+        sha
```

Comment:
If the version file doesn't exist, I'm expecting it to return `nil` so that we can fall back to `git_ref` up above.